### ISSUE TITLE
Access denied on POST xhr

### ISSUE THREAD

**Issue by kpetrianakis** [altso/WebViewAjaxInterceptor#2]
Hello, this is great to learn how to intercept xhr on UWP.

However, POST requests give me an access denied error. Are they supposed to work for you?

Thanks

**Comment by altso** [comment]
@kpetrianakis no, POST didn't work for me as well. Anyways, you won't be able to access the body of the request as `IUriToStreamResolver` doesn't provide this info. We came up with an interceptor on the client side (JS) that converts all POST requests to GET and puts a body in a query string as a parameter. Obviously, it's not a perfect workaround, but we were able to use that.

**Comment by kpetrianakis** [comment]
Thanks for the quick reply mate.
I had the same idea... I will implement it soon. I also voted up the feature of getting the POSTs in the future.
Have you come across any other workaround?

**Comment by altso** [comment]
I believe you can run an http proxy for you application and intercept whatever you need. I haven't tried that though.

**Comment by kpetrianakis** [comment]
What do you mean? An HTTP server inside the app that will serve the local pages and will request them by HttpClient?
Or what else?

**Comment by altso** [comment]
Yes, I meant local HTTP server.
